### PR TITLE
Fix unmarshal block response

### PR DESCRIPTION
**Pull request by yrong**
Resolves: SNO-770

### CODE REVIEW

File: relayer/relays/beacon/header/syncer/api/api_response_test.go [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+import (
+	"encoding/json"
+	"github.com/stretchr/testify/assert"
+	"os"
+	"testing"
+)
+
+func TestUnmarshalBlockResponse(t *testing.T) {
+	blockResponse := BeaconBlockResponse{}
+	data, _ := os.ReadFile("web/packages/test/testdata/beacon_block_6815804.json")
```
Comment (claravanstaden) [inline relayer/relays/beacon/header/syncer/api/api_response_test.go:12]: Did you try block `6815803` as well? Were you able to reproduce the error?

Comment (yrong) [reply relayer/relays/beacon/header/syncer/api/api_response_test.go:12]: 6815803 is fine.

Comment (claravanstaden) [reply relayer/relays/beacon/header/syncer/api/api_response_test.go:12]: Nice catch! 👏🏻

**Review by claravanstaden — APPROVED** [review]

**Review by alistair-singh — APPROVED** [review]
Nice Work!

**Review by vgeddes — APPROVED** [review]
+1

**Pull request merged**